### PR TITLE
MKT-953 -- Add stats engine to navs, update stats metadata

### PR DESCRIPTION
**Pull request by stewsmith**
[MKT-953](https://optimizely.atlassian.net/browse/MKT-953)

* Add link to /statistics page to the Products dropdown menu as: "Stats Engine"
* Add link to /statistics to the site footer, under Product features and benefits menu as: "Stats Engine"
* Update meta info on page:
- seo_title: "Optimizely Stats Engine" 
- og_title: "Optimizely Stats Engine" 
- og_description: "Statistics for the Internet age is here."

### CODE REVIEW

**Comment by robinpam** [comment]
@stewsmith Did you change the og tags and title tag here? Should I not update those in mine?

**Comment by stewsmith** [comment]
@robinpam Yup. Updated in this PR so you don't have to change them in yours.

**Comment by robinpam** [comment]
Ok, thanks Stew

File: website/statistics/index.hbs [hunk 1]
```diff
@@ -8,8 +8,11 @@ body_class:
 - statistics-page
 page_script: statistics.js
 footer_style: grey
-og_title: ""
-og_description: ""
+seo_title: "Optimizely Stats Engine"
```
Comment (nchilada) [inline website/statistics/index.hbs]: `seo_title` is already defined on line 3.  (Though the definition here takes precedence.)  Can you delete one of them?

**Comment by kylerush** [comment]
@stewsmith this pr looks funky. doesn't look like the og_title or og_description are added correctly. they shouldn't be a list as you can't have more than one value.

**Comment by stewsmith** [comment]
@kylerush This is consistent with every other og_title and og_description. I talked to @dtothefp about it and he said that it expects an array and doesn't work with just a string value.

**Comment by kylerush** [comment]
@stewsmith @dtothefp that is incorrect. if you look inside of website-guts/templates/layouts/wrapper.hbs you'll see this:

```html
<meta name="description" content="{{#if page.description}}{{page.description}}{{else}}Deliver your best customer experiences at every touchpoint on the web and mobile apps.{{/if}}">

<title>{{#if page.seo_title}}{{page.seo_title}}{{else}}Optimizely: Make every experience count{{/if}}</title>
```

those do not expect arrays and it doesn't make sense to have more than one description or title tag.

can you update por favor?

**Comment by stewsmith** [comment]
@kylerush Sure, I'll change the rest too.

**Pull request merged**